src/text_shaper/open_shaper.cpp

```diff
@@ -106,9 +106,9 @@ struct hash<FontPathAndSize>
 namespace text
 {
 
-using HbBufferPtr = unique_ptr<hb_buffer_t, void (*)(hb_buffer_t*)>;
-using HbFontPtr = unique_ptr<hb_font_t, void (*)(hb_font_t*)>;
-using FtFacePtr = unique_ptr<FT_FaceRec_, void (*)(FT_FaceRec_*)>;
+using hb_buffer_ptr = unique_ptr<hb_buffer_t, void (*)(hb_buffer_t*)>;
+using hb_font_ptr = unique_ptr<hb_font_t, void (*)(hb_font_t*)>;
+using ft_face_ptr = unique_ptr<FT_FaceRec_, void (*)(FT_FaceRec_*)>;
```

Comment:
These changes shouldn't be done in this PR, since this PR only meant to change crispy and it's related stuff. Maybe we can have this in your follow-up renaming PR for other modules

Reply:
I will make then separate PR only for open_shaper part

Reply:
Well I didn't notice earlier but text_shaper seems to already follow current cripsy naming scheme, so maybe commit this change and it's relted parts in different commit in the same this PR.

Reply:
This changes caused by .clang-tidy changes to apply naming convention for typeAlias